summarization/summarizer_main.py

```diff
@@ -352,7 +352,7 @@ def _get_stamp_descriptor_for_embedded_content(self, content):
         # are now of type str
         return self.embedded_descriptors_dict[str(content.content_type)]
 
-    def _filter_images_with_text(self):
+    def _filter_images_with_text_and_create_stamps(self):
```

Comment:
`create_stamps_and filter_images_with_text`. Also edit the comment below

Reply:
updated.